linux/jre/alpine/src/main/packaging/temurin/21/APKBUILD

```diff
@@ -60,6 +61,7 @@ _jre() {
 	mv "$_fromroot/conf"    "$_toroot"
 	mv "$_fromroot/legal"   "$_toroot"
 	mv "$_fromroot/lib"     "$_toroot"
+	mv "$_fromroot/man"     "$_toroot"
```

Comment:
@steelhead31 do we know if these were intentionally removed from more recent JRE's?

Reply:
NVM, I've worked out why we skipped it now